### ISSUE TITLE
Text max width in docs

### ISSUE THREAD

**Issue by bartaz** [canonical/juju.is#514]
It seems that recent change removed max width limitation in docs (#487) in order to make it consistent with lists.

It should be fixed the other way around. Max width is there for a reason, to avoid lines of text that are too long to read comfortably.

There is a utility in Vanilla that allows to fix this (limit lists to paragraph text width): https://vanillaframework.io/docs/utilities/text-max-width. Please use it instead. (it can be used directly on list root element, or on any parent wrapper, if markup comes from discourse for example).

**Comment by chillkang** [comment]
@bartaz In docs pages, there is `notification` component and `table` element so `.text-max-width` wouldn't be applied to those, right? I have removed `p` element from .docs #main-content { max-width: 40rem; } and would this be the right way to format the page?
<img width="1357" alt="Screenshot 2024-01-25 at 3 54 00 PM" src="https://github.com/canonical/juju.is/assets/90341644/281c927d-91b4-4e6f-8f0d-471fa26499a5">

**Comment by bartaz** [comment]
@chillkang Yes, as notification and tables are block elements they are expected to be full width, while text elements (including lists) should be limited to 40rem.

As far as I've seen the proposed PR to updated documentation layout also addresses this issue: https://github.com/canonical/juju.is/pull/510
